### PR TITLE
[go] Enable org-babel integration

### PR DESCRIPTION
**Pull request by swaroopch**
Adds https://github.com/pope/ob-go emacs package

### CODE REVIEW

**Comment by swaroopch** [comment]
@bogdanteleaga Can you please review this PR as well? :)

**Comment by bogdanteleaga** [comment]
Hmm, I'm still quite bad at using org mode and I've never used babel before. But there doesn't seem to be much here, if it works I don't see why we wouldn't include it.

**Comment by swaroopch** [comment]
@bogdanteleaga Thanks. You can try it out by typing this in org mode and in normal mode, do `,,` or `ctrl-c ctrl-c`:

```
#+begin_src go :imports "fmt"
  fmt.Println("Hello, 世界")
#+end_src
```

You should see this appear below those lines:

```
#+RESULTS:
: Hello, 世界
```

File: layers/+lang/go/packages.el [hunk 1]
```diff
@@ -96,3 +99,9 @@
       :defer t
       :init
       (push 'company-go company-backends-go-mode))))
+
+
+(defun go/init-ob-go ()
+  (use-package ob-go
+    :if (configuration-layer/layer-usedp 'org)
+    :ensure t))
```
Comment (syl20bnr) [inline layers/+lang/go/packages.el]: why this `:ensure t` ?

Comment (swaroopch) [inline layers/+lang/go/packages.el]: @syl20bnr You're right, the `:ensure t` is not needed, removing it in the next push to the branch.

P.S. Sorry for the delayed response!

**Comment by swaroopch** [comment]
@syl20bnr The `:ensure t` has been removed. Can you please review?

File: layers/+lang/go/packages.el [hunk 2]
```diff
@@ -5,6 +5,9 @@
         flycheck
         go-mode
         go-eldoc
+        (ob-go :location (recipe
+                          :fetcher github
+                          :repo "pope/ob-go"))
```
Comment (joehillen) [inline layers/+lang/go/packages.el]: ob-go is on MELPA now

https://melpa.org/#/ob-go

**Comment by swaroopch** [comment]
I made this PR exactly one year ago, since there doesn't seem to be interest in this, I'm closing this PR, if anybody is up for it, please feel free to take it forward.

**Pull request closed without merging**